docs/content/token/staking/stake-slashing.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Stake Slashing
+description: How the Flow protocol punishes bad behavior be slashing stakers tokens
```

Comment:
```suggestion
description: How Flow enforces honest node behaviour
```

Feel free to ignore or reword this suggestion. The description field is used in link previews and therefore is often truncated, so I try to keep them a bit shorter